spec/lib/health_monitor/providers/delayed_job_spec.rb

```diff
@@ -59,37 +106,4 @@
       }.to change { described_class.new.configuration.queue_size }.to(queue_size)
     end
   end
-
-  describe '.check_failed_latency' do
```

Comment:
you wanted to test `check` as above, not `check_failed`

Reply:
oh ok. this is the part I was beginning to wonder if I was doing wrong since `check_failed_latency` is a private method